about.html

```diff
@@ -0,0 +1,129 @@
+          <nav class="main-nav">
+              <div class="icon-nav">
+                  <a href="index.html"><img src="./images/mylogo.png" alt="logo image"></a>
+              </div>
+              <div class="icon-menu-mobile">
+                  <img src="images/icons/hamburger-icon.svg" alt="Hamburger icon">
+              </div>
+              <ul class="bottom-nav">
+                  <li><a  class="active" href="about.html">About</a>
+                  <li><a href="#Sales">Sales</a></li>
+                  <li><a href="#Partners">Partners</a></li>
+                  <li><a href="#Testimonials">Testimonials</a></li>
+                  <li><a href="#">Blog</a></li>
+                  <li><button type="button">Follow US</button></li>
+              </ul>
+          </nav>
+      </div>
+  </div>
+
+  <div class="nav-mobile">
+      <div class="cross">
+          <img src="images/icons/cross.svg" alt="cross">
+      </div>
+      <ul class="container">
+          <li><a href="about.html">About</a>
+          <li><a href="#program">Services</a></li>
+          <li><a href="#testimonials">Testimonials</a></li>
+          <li><a href="#partners">Partners</a></li>
+          <li><a href="#">Blog</a></li>
+      </ul>    
+  </div>
+  </header>
+   
+  <div id="about-container1">
+    <p class="about-description">This is Lugard's Tech Development Academy '3ple Tech'</p>
+    <hr class="about-description-hr">
+    <h1 class="about-header">AT 3PLE TECH OUR DEDICATION IS BASED ON BEST PRACTICES AND CLIENTS SATISFACTION.</h1>
+    <div id="about-theme">
+      <p class="about-theme">
+        3ple Tech is service delivery and learning based academy, with a very 
+        vast network of resource persons and professional developers. Our team comprises of dedicated
+        workers and effective code developers. Our objective is to bridge knowledge-based gap of the tech 
+       industry especially from the African perspective.
+      </p>
+    </div>
+    <p class="about-contact">
+      Please contact us via email or phone call for further enquiry to 3ple Tech 
+      and service delivery.
+    </p>
+    <p class="about-contact-mail">agulugard@gmail.com</p>
+    <hr class="about-contact-hr">
+  </div>  
+  
+  <div class="about-logo">
+    <div class="about-logo2">
+      <img src="./images/mylogo.png" class="about-mylogo" alt="mylogo">
+      <p class="academy-description">3ple Tech academy<br>October 2022<br>Enugu, Biafra.</p>
+    </div>
+  </div>
+  <div id="about-members">
+    <h3>Team members</h3>
+    <p class="about-members-description">Take a look at some of the 3ple Tech academy software Developers and instructors</p>
+    <ul>
+      <li><img src="images/grapy-picture1.png" class="about-pic" alt="illustrative-pic"></li>
+      <li><img src="images/grapy-picture2.png" class="about-pic" alt="illustrative-pic"></li>
+    </ul>
+  </div>
+
+  <div id="about-partners">
+    <div class="about-wrapper-partners">
+     <h2>Partners</h2>
```

Comment:
you are required to replicate a design similar to the provided design 👍🏾  however your design does nit follow the required design.

### your design 
![image](https://user-images.githubusercontent.com/60612329/198721612-afa1e556-b641-46d2-a1ee-371ceb3fcf07.png)


### required design
![image](https://user-images.githubusercontent.com/60612329/198721934-6c6225f6-04f8-4426-86b3-db7009a6ed4c.png)